src/basicOp.js

```diff
@@ -24,7 +24,24 @@
  * @return {number}
  */
 function basicOp(operation, value1, value2) {
-  // write code here
+  let result;
+  switch (operation) {
+    case '+':
+      result = value1 + value2;
```

Comment:
will be better to write like this
```suggestion
      return value1 + value2;
```
and now you no need `break`